/*
Abstract:
  OGG settings widget

Last changed:
  $Id$

Author:
  (C) Vitamin/CAIG/2001

  This file is a part of zxtune-qt application based on zxtune library
*/

#pragma once
#ifndef ZXTUNE_QT_UI_OGG_SETTINGS_H_DEFINED
#define ZXTUNE_QT_UI_OGG_SETTINGS_H_DEFINED

//local includes
#include "backend_settings.h"

namespace UI
{
  BackendSettingsWidget* CreateOGGSettingsWidget(QWidget& parent);
}

#endif //ZXTUNE_QT_UI_OGG_SETTINGS_H_DEFINED
